Reading List/Reading List/BookTableViewCell.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  BookTableViewCell.swift
+//  Reading List
+//
+//  Created by Drew Miller on 5/19/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class BookTableViewCell: UITableViewCell {
+    //Outlets
+    @IBOutlet weak var bookLabelOutlet: UILabel!
+    @IBOutlet weak var readBookCheckedOutlet: UIButton!
+   //Action
+    @IBAction func readBookCheckedAction(_ sender: Any) {
+        func toggleHasBeenRead(for: BookTableViewCellDelegate) {
+    }
+
+    
+var book: Book?
```

Comment:
This is going to be a computed property that uses a didSet and calls the updateViews method